client/components/Coverages/CoverageAddAdvancedModal.tsx

```diff
@@ -206,7 +212,7 @@ class CoverageAddAdvancedModalComponent extends React.Component<IProps, IState>
                     desk: get(coverage, 'desk._id'),
                 };
 
-                if (coverage.coverage_id != null && !coverage.enabled) {
+                if (coverage.coverage_id != null && coverage.enabled === false) {
```

Comment:
when dealing with code that is not typed well, it's better to do `coverage.enabled !== true` to ensure `undefined` is treated as false.